setup.py

```diff
@@ -89,6 +89,7 @@ def get_version(filename="jrnl/__init__.py"):
         "keyring>=7.3",
         "passlib>=1.6.2",
         "pyxdg>=0.25",
+        "asteval>=0.9.8",
```

Comment:
No, this is probably a great way to handle it. Great catch.

(If anyone else is following up on this, `template.py` has been moved to [jrnl/plugins/template.py](https://github.com/maebert/jrnl/blob/2.0-rc1/jrnl/plugins/template.py#L2))